ext/bg/js/translator.js

```diff
@@ -45,9 +46,113 @@ class Translator {
             await this.buildTermFrequencies(definition, titles);
         }
 
+        if (options.general.compactTags) {
+            for (const definition of definitionsGrouped) {
+                dictTermsCompressTags(definition.definitions);
+            }
+        }
+
         return {length, definitions: definitionsGrouped};
     }
 
+    async findTermsMerged(text, dictionaries, alphanumeric) {
+        const options = await apiOptionsGet();
+        const mainDictionary = Object.keys(options.dictionaries).filter(dict => options.dictionaries[dict].main).concat([''])[0];
+        const secondarySearchTitles = Object.keys(options.dictionaries).filter(dict => options.dictionaries[dict].allowSecondarySearches);
+        const titles = Object.keys(dictionaries);
+        const {length, definitions} = await this.findTerms(text, dictionaries, alphanumeric);
+
+        const definitionsBySequence = dictTermsMergeBySequence(definitions, mainDictionary);
+
+        const definitionsMerged = [];
+        const mergedByTermIndices = new Set();
+        for (const sequence in definitionsBySequence) {
+            if (!(sequence >= 0)) {
```

Comment:
That's a pretty strange way of saying `if (sequence < 0)`...

Reply:
Huh, I wonder what I was thinking there :D I even modified that afterwards due to an off-by-one error caused by confusion over whether the sentinel value should be `0` or `-1`.